web/beacon-app/src/components/layout/Topbar/ScheduleOfficeHours.tsx

```diff
@@ -0,0 +1,15 @@
+import { Trans } from '@lingui/macro';
+import { Button } from '@rotational/beacon-core';
+import { Link } from 'react-router-dom';
+
+const ScheduleOfficeHours = () => {
```

Comment:
Would you mind moving the ScheduleOfficeHours components to the root of `components` folder ?